### DISCUSSION TITLE
Boat launch types

### DISCUSSION THREAD

**Discussion by Parel803** [PygmalionOfCyprus/cmo-db-requests#3896] [Q&A]
Is there in the DB a difference for a ship to launch a RHIB, FRISC, etc. Like davits, crane of slipway?
regards GJ

**Comment by claudejdev** [comment]
> Is there in the DB a difference for a ship to launch a RHIB, FRISC, etc. Like davits, crane of slipway? regards GJ

Hey there @Parel803 
you might want to take a look at the DataShipDockingFacilities table
ID refers to the Ship's ID (in DataShip), ComponentNumber is the order on that ship, ComponentID refers to the DockingFacility's ID in the DataShipDockingFacility table.

**Comment by Parel803** [comment]
thank you, I'll check
regards GJ

**Comment by PygmalionOfCyprus** [comment]
Currently we don't distinguish between davits, ramps, etc., we just distinguish between size. So, a large LCVP couldn't be launched from davits for a small RHIB or something.

**Comment by Parel803** [comment]
Oke, thanks for the answer.
regards GJ
